sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/expressions/json/JsonExpressionEvalUtils.scala

```diff
@@ -58,11 +59,11 @@ object JsonExpressionEvalUtils {
 }
 
 class JsonToStructsEvaluator(
```

Comment:
shall we make it a case class?

Reply:
Yeah, this idea is `great` and can `simplify` code. 
Thank you very much!